Assets/Scripts/Gamestate/PlayerManager.cs

```diff
@@ -69,15 +69,15 @@ void TurnIntoRagdoll(Vector3 impactSite, Vector3 impactDirection)
         // Disable components
         GetComponent<PlayerMovement>().enabled = false;
         healthController.enabled = false;
-        meshBase.SetActive(false);
         // TODO display guns falling to the floor
         gunController.gameObject.SetActive(false);
         // Disable all colliders and physics
-        GetComponents<Collider>().ToList().ForEach(collider => collider.enabled = false);
-        GetComponent<Rigidbody>().useGravity = false;
         // Ragdollify
-        ragdoll.gameObject.SetActive(true);
-        ragdoll.AddForceAtPosition(impactDirection * 4, impactSite, ForceMode.Impulse);
+
+        // TODO: Make accurate hitbox forces for the different limbs of the player
+        //ragdoll.AddForceAtPosition(impactDirection * 4, impactSite, ForceMode.Impulse);
```

Comment:
and pls delete this commented line since we're adding sth different next time and git has history